README.md

```diff
@@ -635,6 +635,8 @@ The `config` command accepts the following subcommands:
            (Using `$EDITOR`, e.g: `export EDITOR=/usr/bin/emacs`)
 -    `delete` removes the specified configuration
 -    `migrate` is a command used to change the configuration format between versions of `pgenv`
+-    `path` accepts a version number and prints on standard output the path to such version
```

Comment:
```suggestion
-    `path` accepts a version number (or "default") and prints on standard output the path to such version
```

Reply:
Well, every `config` command accepts `default`, I think it is already clear, but I added as you suggest.